Jellyfin.Plugin.TMDbBoxSets/Jellyfin.Plugin.TMDbBoxSets.csproj

```diff
@@ -1,14 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
     <IsPackable>true</IsPackable>
     <AssemblyVersion>4.0.0</AssemblyVersion>
     <FileVersion>4.0.0</FileVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Jellyfin.Controller" Version="10.*" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.5.0-pre1" />
```

Comment:
Not sure if we need to wait for a proper release to update this, I couldn't get `10.5.*` to work for a version number.

Reply:
Try just 10.*, that should work and is what we did for 10.4.0.

Reply:
Yeah that's what I tried initially but it seemed to keep pulling 10.4.0, I couldn't get it to pull 10.5 until I specified the full version. It's possible there's a better way but a quick glance at the docs didn't show anything and I checked a couple other plugin implementations and they had the same 10.*

Reply:
10.* works now that 10.5 is in NuGet! I reverted this change